### ISSUE TITLE
Run in classic interpreter mode under fast-interpreter building 

### ISSUE THREAD

**Issue by hungryzzz** [bytecodealliance/wasm-micro-runtime#2164]
Hi, I build wamr with default building options. Is there any way to open the classic interpreter mode(e.g., open execution option) under fast-interpreter building? I saw --interp option in https://github.com/bytecodealliance/wasm-micro-runtime/issues/1842 will add such feature, does it work now?

**Comment by TianlongLiang** [comment]
Hi, CLI options can't be used to switch between classic and fast interpreters. The different and overlapping compilation units are used for those two modes. You may find more details about running modes in [this blog](https://bytecodealliance.github.io/wamr.dev/blog/introduction-to-wamr-running-modes/#compile-the-engines). .

TL;DR: the fast interpreter mode can't coexist with other modes. If you want to use fast interpreter mode, you have to build it using those commands (on Linux):

```sh
cd product-mini/platforms/linux/
mkdir build
# default CMake config build with fast interpreter mode, unlike in release, iwasm used multi-tier jit mode
cmake -B build
cmake --build build
```
